### PR TITLE
update and fix scylla configs

### PR DESCRIPTION
**Pull request by kthommandra**
- added new config for scylla-dev (dev cluster for scylla, stats go to
qubit_dev database)
- fixed the namepass to scylla* since namepass is about measurement
names rather than plugin name
- pushed 1.2.0 tag to the aristanetworks/telegraf repo and republish
the new RPMs to ToolsV2

Testing:
1) tested scylla configs on scylla nodes
2) tested update of telegraf config on us161, watched server metrics in
   grafana and it was fine

### CODE REVIEW

File: Arista/ConfigFiles/telegraf-qubit-scylla-dev.conf
Comment (StephenSDaily) [inline Arista/ConfigFiles/telegraf-qubit-scylla-dev.conf:18]: Do you want the precision to be seconds?

**Comment by manojm321** [comment]
where does the tag info show up?

~ @us108.sjc> rpm -qip telegraf-1.2.0-2.x86_64.rpm
Name        : telegraf
Version     : 1.2.0
Release     : 2
Architecture: x86_64
Install Date: (not installed)
Group       : default
Size        : 31270766
License     : MIT
Signature   : (none)
Source RPM  : telegraf-1.2.0-2.src.rpm
Build Date  : Fri 27 Oct 2017 02:26:49 PM PDT
Build Host  : us161.sjc.aristanetworks.com
Relocations : /
Packager    : <kthommandra@us161.sjc.aristanetworks.com>
Vendor      : Influxdata
URL         : github.com/aristanetworks/telegraf
Summary     : InfluxDB Telegraf agent
Description :

**Review by StephenSDaily — APPROVED** [review]
Everything looks okay. Please wait for approval from Manoj before merging as well.

File: Arista/ConfigFiles/telegraf-qubit-scylla-dev.conf
Comment (kthommandra) [reply Arista/ConfigFiles/telegraf-qubit-scylla-dev.conf:18]: the scrape period of these states is the default 15secs and so precision of seconds seems fine. These are not really events from Scylla

**Comment by kthommandra** [comment]
Version : 1.2.0
Release : 2

The version information is picked from the git tag.
It should be noted that the git tag is the tag of the upstream telegraf from which we pulled last
The release shows the changes that were to the binary if any. Since we moved to the new binary RPM content (the default config file), I bumped it up.

So the way to read this info IMHO is - this telegraf binary is based on 1.2.0 tag of upstream telegraf with some changes in the binary RPM by us

hth

**Review by manojm321 — APPROVED** [review]

**Pull request merged**
